src/main/java/scooter_wow.java

```diff
@@ -0,0 +1,29 @@
+/* A public class Scooter that implements the Drivable and Tradable interfaces
+* */
+public class scooter_wow implements Drivable, Tradable {
+    private int maxSpeed;
+
+    public scooter_wow() { this.maxSpeed = 5; }
```

Comment:
Classes should be named using the mixed case convention, therefore the class 'scooter_wow' should be renamed 'ScooterWow' !